.github/dependabot.yml

```
@@ -0,0 +1,31 @@
+version: 2
+registries:
+    gradle-plugin-portal:
+        type: maven-repository
+        url: https://plugins.gradle.org/m2
+        username: dummy # Required by dependabot
+        password: dummy # Required by dependabot
+updates:
+    - package-ecosystem: "gradle"
+      directory: "/"
+      registries:
+          - gradle-plugin-portal
+      schedule:
+          interval: "daily"
+          time: "02:00"
```

To avoid alarm bell at night. 😆
```suggestion
          time: "10:00"
```